sci-biology/open-ephys-gui/open-ephys-gui-0.5.5.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
While you're at it, could you please bump the EAPI of this ebuild? I don't expect any issues with EAPI 8 for this ebuild so it should be trivial in this case.

Reply:
I tried, but then for reasons I do not understand, with `EAPI=8` cmake will try to build with an invalid build type. I could not get it to pass the configure phase with `EAPI=8`.